src/sync.rs

```diff
@@ -190,7 +226,32 @@ impl<A> SyncContext<A>
 where
     A: Actor<Context = Self>,
 {
-    /// Create new SyncContext
+    /// Create new SyncContext. This is used insted of impl Actor for your Actor
```

Comment:
I just noticed, `SyncContext::new` is not public, there is no need to add docstring for it